dask/highlevelgraph.py

```diff
@@ -251,6 +252,9 @@ def __init__(self, mapping, dependencies=None, global_dependencies=None):
         self.global_dependencies = global_dependencies
         self.global_dependencies_has_been_trimmed = False
 
+        annotations = config.get("annotations", None)
+        self.annotations = None if annotations is None else annotations.copy()
```

Comment:
Maybe this should move to a new `Layer.__init__` function and all layer subclasses should call `super`?  cc also @madsbk

Reply:
Also, thoughts on this?  It's a purely aesthetic change and not very important.

```suggestion
        self.annotations = copy.copy(config.get("annotations", None))
```

Reply:
Agreed, done

Reply:
> Maybe this should move to a new `Layer.__init__` function and all layer subclasses should call `super`? cc also @madsbk

That is fine with me.